core/src/main/scala/chisel3/internal/firrtl/IR.scala

```diff
@@ -242,7 +243,7 @@ sealed case class UnknownWidth() extends Width {
   override def toString: String = ""
 }
 
-sealed case class KnownWidth(value: Int) extends Width {
+sealed private[chisel3] case class KnownWidth(value: Int) extends Width {
```

Comment:
```suggestion
sealed case class KnownWidth(value: Int) extends Width {
```
This also needs to remain public since `.widthOption` returns `Width` which could be a `KnownWidth` (see used by chiseltest: https://github.com/ucb-bar/chisel-testers2/blob/b5a34f18be52d7879ffbdb0468c724a4d539f900/src/main/scala/chiseltest/iotesters/PeekPokeTester.scala#L123)